JavaScript/Topics/18.Rollovers/Rollovers.md

```diff
@@ -0,0 +1,162 @@
+
+  * ### You react to the onmouseover event by using the __onmouseover__ attribute of an HTML tag that defines the object on the web page and then assign to the onmouseover attribute the action you want performed when the event occurs. 
+   * ### The action can assign a new value to an attribute of an object, call a method of an object, or call a JavaScript function.
+
+
+
+## **Example 1:**
+<br>
+
+### In example below, we create a rollover effect that can change the color of its text using the style attribute.
+
+```html
+<p
+onmouseover=”this.style.color=’red'”
+onmouseout=”this.style.color=’blue'”>
+Move the mouse over this text to change its color to red. Move the mouse away to
+change the text color to blue.
+</p>
+```
+
+## Output
+
+<br>
+
+![image](op-1.png)
+<br><br>
+
+
+<br>
+
+![image](op-2.png)
+<br><br>
+
+
+## **Example 2:**
+ ### This example shows how to create rollover effect that involves text and images. When the user places his or her mouse pointer over a book title, the corresponding book image appears.
+
+ ```html 
+
+<html>
+<head>
+<title>Rollover Effect</title>
+</head>
+<body>
+<table>
+<tbody>
+<tr valign=”top”>
+<td width=”50″>
+<a><img src=”vb2010book.jpg” name=”book”></a>
+</td>
+<td><img height=”1″ width=”10″></td>
+<td><a onmouseover=”document.book.src=’vb2010book.jpg'”><b>Visual Basic 2010 Made Easy</b></a>
+<br>
+<a onmouseover=”document.book.src=’vb2008book.jpg'”><b>Visual Basic 2008 Made Easy</b></a>
+<br>
+<a onmouseover=”document.book.src=’vb6book.jpg'”><b>Visual Basic 6 Made Easy</b></a>
+<br>
+</td>
+</tr>
+</tbody>
+</table>
+</body>
+</html>
+
+```
+## Output 
+
+<br>
+
+![image](op-3.PNG)
+<br><br>
+<br>
+
+![image](op-4.png)
+<br><br>
+
+<br>
+
+![image](op-5.png)
+<br><br>
+
+
+# **Text Rollover**
+ You can create as many rollovers as you want on your web page; however,each one should be meaningful to the visitor. There is nothing more distracting to a visitor than to encounter rollovers on practically every object on a web page. Carefully placed rollovers can enhance a visitor's experience when browsing the web page. A clever rollover technique used by some developers is to enable a visitor to see additional information about an item described in text by placing the mouse cursor on the text. This eliminates the time-consuming task of using a hyperlink to display another web page that contains this additional information and reduces the information clutter found on some web pages.You create a rollover for text by using the **onmouseover** attribute.
+
+
+## **Example :**
+
+### In example below, we create a rollover effect that can change the text when move goes over the text.
+
+```html
+<HTML>
+<head></head>
+<Body>
+<textarea rows=”2″ cols=”50″ name=”rollovertext” onmouseover=”this.value=’What is rollover?'”
+onmouseout=”this.value=’Rollover means a webpage changes when the user moves his or her mouse over an object on the page'”></textarea>
+</body>
+</html>
+```
```

Comment:
````suggestion
```html
<HTML>
<head></head>
<Body>
<textarea rows="2" cols="50" name="rollovertext" onmouseover="this.value='What is rollover?'"
onmouseout="this.value='Rollover means a webpage changes when the user moves his or her mouse over an object on the page'"></textarea>
</body>
</html>
```
````

Reply:
Sir, I will change it .Actually this might have happened while creating a file. Sorry for inconvenience :(